### ISSUE TITLE
LICENSE

### ISSUE THREAD

**Issue by zhanghang1989** [donnyyou/torchcv#13]
This is a great project. I found the Synchronized Batch Normalization part is from one of my repo. Please keep the orginal copyright and notice in the LICENSE file:

https://github.com/zhanghang1989/PyTorch-Encoding/blob/master/LICENSE

Please also consider crediting [my repo](https://github.com/zhanghang1989/PyTorch-Encoding). Thanks

**Comment by youdonny** [comment]
Thanks for your great project! I have maintained a thanks list in the private version. And I will update later!!!
